astropy/units/tests/test_format.py

```diff
@@ -542,15 +542,13 @@ def test_vounit_custom():
     assert x_string == 'm mfoo'
 
 
-def test_vounit_implicit_custom():
-    # Yikes, this becomes "femto-urlong"...  But at least there's a warning.
-    with pytest.warns(UnitsWarning) as w:
-        x = u.Unit("furlong/week", format="vounit")
-    assert x.bases[0]._represents.scale == 1e-15
-    assert x.bases[0]._represents.bases[0].name == 'urlong'
-    assert len(w) == 2
-    assert 'furlong' in str(w[0].message)
-    assert 'week' in str(w[1].message)
+def test_vounit_no_more_implicit_custom():
```

Comment:
This test seems to duplicate `test_unsupported_vounit` you've added to `test_units.py`. Is this still needed here?